horovod/runner/launch.py

```diff
@@ -426,10 +453,23 @@ def parse_args():
                                choices=config_parser.LOG_LEVELS,
                                help='Minimum level to log to stderr from the Horovod backend. (default: WARNING).')
     group_logging_timestamp = group_logging.add_mutually_exclusive_group()
-    group_logging_timestamp.add_argument('--log-hide-timestamp', action=make_override_true_action(override_args),
-                                         help='Hide the timestamp from Horovod log messages.')
-    group_logging_timestamp.add_argument('--no-log-hide-timestamp', dest='log_hide_timestamp',
-                                         action=make_override_false_action(override_args), help=argparse.SUPPRESS)
+    group_logging_timestamp.add_argument('--log-with-timestamp', 
+                                         action=make_override_true_action(override_args),
+                                         help=argparse.SUPPRESS)
+    group_logging_timestamp.add_argument('--log-without-timestamp', dest='log_with_timestamp',
+                                         action=make_override_false_action(override_args), 
+                                         help='Hide the timestamp from Horovod internal log messages.')
+    group_logging_timestamp.add_argument('-prefix-timestamp', '--prefix-output-with-timestamp', action='store_true',
```

Comment:
`-prefix-timestamp` with a single `-` because it is the short version? or because one `-` is missing?

Reply:
Yes, this is the short version for people who want to avoid keystrokes.